bower.json

```diff
@@ -23,7 +23,7 @@
     "cf-icons": "^1.0.0",
     "cf-pagination": "^1.0.0",
     "cf-typography": "^1.0.0",
-    "cf-layout": "^1.0.0"
+    "cf-layout": "^2.0.0"
```

Comment:
Only cf-layout moved to 2.x.x due to breaking changes. Everything else is still 1.x.x.